src/components/layout.jsx

```diff
@@ -18,8 +17,6 @@ function Layout({ children, slug }) {
       <Helmet onChangeClientState={(newState) => setPageTitle(newState.title)} />
```

Comment:
The `Helmet` should be available in the other file.

Reply:
I moved it to footer, but I'm wondering why we need to update the helmet title? We already have helmet in our template pages. Would having it here interfere?

Reply:
Can we use the title from our translation file and have it like this:

```
          <div id="feedback-widget" className="footer__feedback-widget" lang={i18n.language} pageTitle={t('title')} endpoint={process.env.GATSBY_ENDPOINT} />

```

Reply:
Also why do we need:  
```
const [pageTitle, setPageTitle] = useState(t('title')); // initial title is header title
```

Reply:
> I moved it to footer, but I'm wondering why we need to update the helmet title? We already have helmet in our template pages. Would having it here interfere?

I use the Helmet to update the state `pageTitle`. Also, don't see why it would interfere with the Helmet in the template pages since I'm not changing anything about the page; I'm only updating the state.

Reply:
> Can we use the title from our translation file and have it like this:
> 
> ```
> <div id="feedback-widget" className="footer__feedback-widget" lang={i18n.language} pageTitle={t('title')} endpoint={process.env.GATSBY_ENDPOINT} />
> ```
> 
> Specifically, use `pageTitle={t('title')}`

I'm not sure if we should use the default title because each page can have a different title and the default title won't be as specific compared to using the page title.

Reply:
> Also why do we need:
> 
> ```
> const [pageTitle, setPageTitle] = useState(t('title')); // initial title is header title
> ```

We need `useState` to update the title for the feedback module for every page change.

Reply:
If we set page title in the markdown, we can use it to set the title in helmet and for the feedback module. I don't think we need `useState` in that case.

Reply:
Also you use the page title for the initial state, I don't see when that state would change. Won't the feedback module be rendered on every page, which are all created at build time?

Reply:
> > I moved it to footer, but I'm wondering why we need to update the helmet title? We already have helmet in our template pages. Would having it here interfere?
> 
> I use the Helmet to update the state `pageTitle`. Also, don't see why it would interfere with the Helmet in the template pages since I'm not changing anything about the page; I'm only updating the state.

But is it okay to use the Helmet component multiple times? Might you be updating the title only on the Helmet in the footer (or used to be in the layout) and not the one in the template page?